code/game/machinery/cloning.dm

```diff
@@ -373,16 +378,19 @@
 		to_chat(occupant, "<span class='warning'><b>Agony blazes across your consciousness as your body is torn apart.</b><br><i>Is this what dying is like? Yes it is.</i></span>")
 		playsound(src.loc, 'sound/machines/warning-buzzer.ogg', 50, 0)
 		occupant << sound('sound/hallucinations/veryfar_noise.ogg',0,1,50)
-		QDEL_IN(occupant, 40)
+		var/obj/item/organ/brain/B = occupant.getorganslot("brain")
+		B.Remove(occupant)
+		B.forceMove(src)
+		occupant.gib(TRUE, TRUE, TRUE)

```

Comment:
I wasn't certain on the brain bit, that's easy enough to change. The main reason I kept that in was because of what I thought was the null space thing, but I've gone back and tested and what I was seeing might be an issue with re-enter-corpse when you don't actually have a corpse. The verb puts you on the arrival shuttle and makes you only see darkness.